packages/gradient-block/src/Block.tsx

```diff
@@ -0,0 +1,24 @@
+import type { FC } from 'react';
```

Comment:
Seems like you didn't used `example` folder as bootstrap, please use it as it contains many configuration (scripts, eslint config, ...)

Reply:
OK, will do! Thanks! ;) 

Btw, the `example` folder wasn't mentioned in [Silvio's loom](https://www.loom.com/share/aace1ffc875e45dca074760dc40e77e8) which we followed when setting up our gradient block.